package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mx-react-components",
-  "version": "3.1.33",
+  "version": "3.1.34",
```

Comment:
We usually don't include version bumps in the PR.  Was there a reason you added it here?  This would cause problems if someone deployed before you and they bumped the version.